Dockerfile

```diff
@@ -28,7 +28,16 @@ RUN wget https://dl.google.com/go/go1.9.4.linux-amd64.tar.gz -qO /tmp/go.tar.gz
 WORKDIR /app
 VOLUME /app
 
-ADD build.sh /
-ENTRYPOINT ["/build.sh"]
+ENTRYPOINT ["/go/bin/go"]
```

Comment:
I would also suggest to add:
```
CMD ["build"]
```
It will still allow to override the command and will provide a sensible default.

Reply:
Added to both Dockerfile as well as docker-compose.yaml